src/index.ts

```diff
@@ -0,0 +1,4 @@
+import * as hathorLib from './lib';
+
+export default hathorLib;
```

Comment:
Why do you need both exports?

Reply:
The current exported hathorlib enables importing as both named import and default import and we use both on the wallets all the time.
It's very common to use any of the import styles below:
```js
import hathorLib from '@hathor/wallet-lib';
import { HathorWallet } from '@hathor/wallet-lib';
import hathorLib, { HathorWallet } from '@hathor/wallet-lib';
```

The issue with ES6 exports is that it does allow for default export and named export in the same file, but not for the same object, meaning we would have to duplicate the object with all exports for the named and default export.
But if we add an intermediate file (i.e. `lib.ts`) we can export the lib exactly like it currently is exported.

Reply:
Apparently this is a design difference between ES6 and CommonJS, because CommonJS (`module.exports`) uses objects as namespaces for modules but it also has to work with objects as default exports.
This is why when importing from ES6 (`import..from` syntax) the CommonJS export works as both named exports and default export, but when exporting from ES6 we have to specify that we want to use both types of exports.